luda/editor/client/src/timeline/clip/ClipComponent.ts

```diff
@@ -51,6 +53,17 @@ export const ClipComponent: Render<
     height: height - 2,
   };
 
+  // TODO: If possible, it should be integrated with ClipComponent.
+  if (isSubtitleClip(clip)) {
+    return SubtitleClipComponent(
+      {
+        clip,
+        timelineState,
+      },
+      props,
+    );
+  }
+
```

Comment:
There is two `clipComponent`. `SubtitleClipComponent` and `ClipComponent`.
It is because `SubtitleClipComponent` is composed of several pathes rather than on rect.
Should we combine them into one `clipComponent`?

Reply:
I added function `renderClip`.
looks https://github.com/NamseEnt/namseent/commit/0763f1f88e6c378eff9e595c69894729119f1903#diff-fc1360505b3444f28e11154cf00638192f1d9987b7c4b136c4de8467175518ed
with https://github.com/NamseEnt/namseent/pull/18/commits/0763f1f88e6c378eff9e595c69894729119f1903